Update extension reference (#741)
mittwald/developer-portal: docs/contribution/50-reference/2-extensions.mdx

            description: "Phone number that may be used for support requests",
            example: "+49 170 123456",
          },
          inherited: {
            type: "boolean",

>             description: "weather the provided description was inherited from the conrtributor",

Reviewer: ```suggestion
            description: "Weather the provided support information was inherited from the conrtributor",
```

        },
        description: "Tags that describe the Extension",
      },
      disabled: {
        type: "boolean",

>         description: "weather the extension is disabled"

Reviewer: ```suggestion
        description: "Weather the extension is disabled"
```

this is the case for all following descriptions. Also: maybe look into rephrasing these. From the name disabled i can guess that this property gives information about the extension beeing disabled - this doesn't explain what `disabled` means.

      },
      assets: {
        type: "array",
        items: {
          type: "object",
          required: ["id", "index", "assetType"],
          properties: {
            id:{
              type: "string",
              format: "uuid",
              description: "The fileId of the asset",
              example: "709049e1-f283-4ef8-a35b-1b2f672852b0"
            },
            index:{
              type: "integer",
              description: "The index of the asset. Does not have to be successive. Can be used to order the assets.",
              example: 1
            },
            assetType:{
              type: "string",
              enum: ["image", "video"],
              example: "image"
            }
          }
        }
      },
      secrets: {
        type: "array",
        items: {
          type: "object",
          properties: {
            secretId: {
              type: "string",
              format: "uuid",
              example: "6b87abc1-f0fd-4518-a21e-7c8bca3c7c25"
            },
            usableUntil:{
              type: "string",
              format: "date-time"
            }
          }
        }
      },

>       frontendFragments: {
>         type: "object",

Reviewer: ```suggestion
      frontendFragments: {
        type: "object",
        description: "This is a map of anchros to urls. The key of the map is the anchor of a frontend fragment.",
```

        additionalProperties: {

>           description:"The Anchor where the Frontend Fragment is to be embedded",

Reviewer: ```suggestion
          description:"The URL of the frontend fragment and additional properties for display in the mStudio.",
```

          type: "object",
          properties: {
            "additionalProperties": {
              type: "object",
              additionalProperties: {

>                 type: "string"

Reviewer: ```suggestion
                type: "string"
                description:"Some Information needed for the mStudio like the icon.",
```

              },
            },
            url: {
              type: "string",
              format: "uri",

>               example: "The Url of the Frontend Fragment"

Reviewer: ```suggestion
              example: "The URL of the frontend fragment."
```

            }
          }
        }
      },
      extensionStatistics: {
        type: "object",
        properties: {
          amountOfInstances: {
            type: "integer",
            example: "42",
            description: "The amount of instances for this extension. Accurate for the Contributor. Publicly rounded to the next lower hundred."
          }
        }
      },
      pricing: {
        type: "array",
        items: {
          type: "object",
          required: [ "priceInCents", "variantKey" ],

>           properties: {

Reviewer: description and isBooingStopped (becomes available in the near future)

Author: wurden ergänzt

            priceInCents: {
              type: "integer",
              example: "5000",

>               description: "Price of the extension in Cents"

Reviewer: ```suggestion
              description: "Price of the extension in Cents."
```

            },
            variantKey: {
              type: "string",

>               description: "indentifiyer of the pricing variant"

Reviewer: ```suggestion
              description: "Identifier of the pricing variant."
```

            },
            name: {
              type: "string",

>               description: "name of the pricing variant"

Reviewer: ```suggestion
              description: "Name of the pricing variant."
```
